Web/starter_project_vanguard/pages/blog/[id].tsx

```diff
@@ -0,0 +1,81 @@
+import Image from 'next/image'
+import { Blog } from '../../types/blog/blog'
+import blogs from '../../data/blogs.json'
+import { useRouter } from 'next/router'
+import Related from '@/components/blog/RelatedBlog'
+
+const SingleBlog = () => {
+  const router = useRouter()
+  let { id } = router.query
+
+  const blog = blogs.filter((item: Blog) => item._id === id)[0]
+  
+  return (
+    <div className="w-3/4 m-auto">
+      <div className="text-center mt-10 flex-col">
+        <h1 className="text-4xl font-bold font-poppins">
+          {blog.title}
+        </h1>
+
+        <p className="mt-4 tracking-widest font-montserrat">
+          PROGRAMMING, TECH | {blog.time} MIN READ
```

Comment:
I think the blog tags should also come from the blog object itself, rather than hardcoded like this.